plugins/available/sudo.plugin.bash

```diff
@@ -0,0 +1,15 @@
+cite about-plugin
+about-plugin 'Toggle sudo at the beginning of the current or the previous command'
+
+function sudo-command-line() {
```

Comment:
Please add the missing function meta header information to this function: `about` and `group`.

See here for an example: https://github.com/Bash-it/bash-it/blob/f2c07f5143b61c3fdb549bfe3012a684d558ba6d/plugins/available/fzf.plugin.bash#L18-L21